WalletWasabi.Fluent/Views/Settings/GeneralSettingsTabView.axaml

```diff
@@ -9,51 +9,54 @@
              x:CompileBindings="True"
              x:DataType="settings:GeneralSettingsTabViewModel">
   <StackPanel Spacing="20" MinWidth="360" MaxWidth="650" HorizontalAlignment="Left" Orientation="Vertical">
-          <StackPanel.Styles>
-            <Style Selector="DockPanel > TextBlock">
-              <Setter Property="Margin" Value="0 8 20 0" />
-              <Setter Property="VerticalAlignment" Value="Top" />
-              <Setter Property="(DockPanel.Dock)"  Value="Left" />
-            </Style>
-            <Style Selector="DockPanel > ToggleSwitch">
-              <Setter Property="Margin" Value="0 0 -12 0" />
-              <Setter Property="HorizontalAlignment" Value="Right" />
-              <Setter Property="OnContent" Value="{x:Null}" />
-              <Setter Property="OffContent" Value="{x:Null}" />
-            </Style>
-          </StackPanel.Styles>
+    <StackPanel.Styles>
+      <Style Selector="DockPanel > TextBlock">
+        <Setter Property="Margin" Value="0 8 20 0" />
+        <Setter Property="VerticalAlignment" Value="Top" />
+        <Setter Property="(DockPanel.Dock)"  Value="Left" />
+      </Style>
+      <Style Selector="DockPanel > ToggleSwitch">
+        <Setter Property="Margin" Value="0 0 -12 0" />
+        <Setter Property="HorizontalAlignment" Value="Right" />
+        <Setter Property="OnContent" Value="{x:Null}" />
+        <Setter Property="OffContent" Value="{x:Null}" />
+      </Style>
+    </StackPanel.Styles>
 
-          <DockPanel>
-            <TextBlock Text="Dark Mode"/>
-            <ToggleSwitch IsChecked="{Binding DarkModeEnabled}" />
-          </DockPanel>
+    <DockPanel>
+      <TextBlock Text="Dark Mode" />
+      <ToggleSwitch IsChecked="{Binding DarkModeEnabled}" />
+    </DockPanel>
 
-          <DockPanel>
-            <TextBlock Text="Auto Copy/Paste Bitcoin addresses" />
-            <ToggleSwitch IsChecked="{Binding AutoCopy}" />
-          </DockPanel>
+    <DockPanel>
+      <TextBlock Text="Start on OS startup" />
```

Comment:
The grandma users won't know what does `OS` means. Rename this?
`Run on startup`
`Start with the system`
or something like this, how other software call this?

Reply:
Will do this in a moment.